lib/index.js

```diff
@@ -41,7 +41,6 @@ GA.prototype.initialize = function(){
 
 GA.prototype.track = proxy('track');
 GA.prototype.identify = proxy('identify');
```

Comment:
Unrelated to the PR, but should we remove the identify proxy? I don't see it in either classic or universal.